docker-compose.yml

```diff
@@ -1,16 +1,6 @@
 version: '3.1'
 services:
 
-  redis-vsip-admin:
```

Comment:
No need to have Redis running locally in Docker now - it uses in-memory store for local dev.